Login.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
@lsimpson667 When this file gets converted to ejs, remember to lowercase the l in login.ejs